### DISCUSSION TITLE
Can we control view/read access for folders inside a GitHub Repository?

### DISCUSSION THREAD

**Discussion by keerthishankar679** [community/community#29002] [Other Feature Feedback, Questions, & Ideas]
Can we control view/read access for folders inside a GitHub Repository?

**Comment by airtower-luna** [comment]
No. If someone has (read) access to a Git repository, they have access to all of it.

**Reply by keerthishankar679** [reply]
There is no way right?

**Reply by airtower-luna** [reply]
None at all, it's part of the design of Git. If someone clones a repository they get *all* of it. Hiding specific things on the web UI wouldn't help at all.
